### ISSUE TITLE
All applications open on the same desktop at startup

### ISSUE THREAD

**Issue by sebasijan** [eksime/VDesk#96]
I have placed the following bat file in `shell:startup`

    vdesk create:3
    vdesk on:1 run:"C:\Program Files\Microsoft Office\root\Office16\OUTLOOK.EXE"
    vdesk on:2 run:%LOCALAPPDATA%\Microsoft\Teams\current\Teams
    vdesk on:3 run:%LOCALAPPDATA%\slack\slack

But at startup, all 3 applications open up on the same desktop (Desktop 3)

Am I doing something wrong?

**Comment by cruzer619** [comment]
nope.. your just missing a timeout between each line..  This is my batch file.

@echo off
vdesk.exe on:1 run:"C:\Program Files (x86)\Microsoft Office\Office16\Outlook.exe"
timeout 1
vdesk.exe on:2 noswitch:true run:"C:\Program Files (x86)\Mozilla Firefox\firefox.exe"
timeout 1
start "C:\Program Files (x86)\Microsoft\Remote Desktop Connection Manager\RDCMan.exe" 
timeout 1
start "" C:\Users\username\AppData\Local\WhatsApp\Update.exe --processStart WhatsApp.exe
